MDANSE/Src/MDANSE/Framework/Jobs/IJob.py

```diff
@@ -141,6 +141,8 @@ def __init__(self, trajectory_input="mdanse"):
         self.outputQueue = Queue()
         self.log_queue = Queue()
 
+        self.run_count = 0
```

Comment:
This should probably just be a `bool` for now, or check if `self._status.state` is set?